### PR TITLE
Task/dtuw 116 tips for filling taxes

### PR DESCRIPTION
**Pull request by Bithika-Redapple**
### What Does This PR Do?
tips for filing taxes section is enabled in dashboard
user account activity section in profile

### Where Should The Reviewer Start?
personal-info component for user account activity
tax-prepare-profile component for tax tips 

### Screenshots
Screenshot from 2020-10-29 12-17-47
Screenshot from 2020-10-29 12-18-16

### Checklist (if applicable):
[ ] item 1
[ ] item 2
[ ] item 3
[ ] item 4
[ ] item 5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,6 +124,25 @@ g stands for global installation. If you use -g later you can use the CLI in any
  1. Create marketing-preferences component .To do that ,we need to use the following command :ng g c marketing-preferences
  2. User marketing preferences is figure out by this tool.
 
+ <a name="account-activity"></a>
+### For Account Activity
+
+ 1. UPdate personal-info component .To do that ,we need to use the following command :ng g c personal-info
```
Comment (drxeno02) [inline README.md:130]: 👍

File: src/app/auth/auth.service.ts [hunk 2]
```diff
@@ -105,7 +106,8 @@ export class AuthService {
       let payloadObj;
      if(data.phone){
           payloadObj={
-          deviceType:"web",
+          deviceType:"Web",
+          //deviceType:"web",
```
Comment (drxeno02) [inline src/app/auth/auth.service.ts:110]: @Bithika-Redapple Remove any unused code. No reason to have this commented out line here.

File: src/app/layout/footer/footer.component.html [hunk 3]
```diff
@@ -21,145 +21,56 @@
...
+			<p>Digitax</p>
 			<ul>
-				<li><a href="">All Digitax mobile apps</a></li>
-				<li><a href="">Official Digitax mobile app</a></li>
-				<li><a href="">TaxCaster tax calculator</a></li>
-				<li><a href="">ItsDeductible donation tracker app</a></li>
+				<li><a routerLink="/signin">Signin</a></li>
+				<li><a routerLink="/signup">Signup</a></li>
+				<li><a routerLink="/privacy-policy">Privacy Statement</a></li>
+				<li><a routerLink="/terms-service">	Terms of Service</a></li>
```
Comment (drxeno02) [inline src/app/layout/footer/footer.component.html:42]: @Bithika-Redapple You can remove the extra, unnecessary space in front of Terms of Service

File: src/app/personal-info/personal-info.component.ts [hunk 4]
```diff
@@ -457,4 +483,18 @@ this.showLastTax = true;
 
 }
 
+//https://s3.amazonaws.com/digitaxbucket/1603178195073-all_data.zip
```
Comment (drxeno02) [inline src/app/personal-info/personal-info.component.ts:486]: @Bithika-Redapple If you want to make a comment regarding the download link, form a full comment. For example,

"Method is used to download tax information from s3. The download link is, https://s3.amazonaws.com/digitaxbucket/1603178195073-all_data.zip"

File: src/app/survey/survey.component.ts [hunk 5]
```diff
@@ -4,114 +4,139 @@ import { FormGroup, FormBuilder, FormArray, FormControl } from '@angular/forms';
...
+this.serveyService.listSurvey().subscribe(categories => {           
+this.categories= categories["data"]; 
+})
+
+}
+
+/*
+* This function is used while change category 
+*/
+
```
Comment (drxeno02) [inline src/app/survey/survey.component.ts:89]: @Bithika-Redapple You have a line break here. This creates a lingering comment. Lingering comments are not attached to anything. Remove the empty line space (on line 89) to attach the comment to the function.

**Review by drxeno02 — APPROVED** [review]
@Bithika-Redapple LGTM! My critiques are
1. Code is inadequately documented (every method isn't commented, lack of inline comments, ect)
2. Code is incorrectly documented. Not using the appropriate types of comments /** vs //
3. Code needs cleanup. There is a lot of unused commented code that should be deleted
4. Code is not formatted. You should run some sort of code formatting tool so that your coding structure and style is organized and easy to read

File: src/app/auth/auth.service.ts (hunk 2 above)
Comment (Bithika-Redapple) [reply src/app/auth/auth.service.ts]: done

File: src/app/layout/footer/footer.component.html (hunk 3 above)
Comment (Bithika-Redapple) [reply src/app/layout/footer/footer.component.html]: it is done

File: src/app/personal-info/personal-info.component.ts (hunk 4 above)
Comment (Bithika-Redapple) [reply src/app/personal-info/personal-info.component.ts]: modified

**Pull request merged**